### ISSUE TITLE
Expected "##TX" or "##MD" block @0x208 but found "b'##CC'"

### ISSUE THREAD

**Issue by tistelac** [danielhrisca/asammdf#140]
# Python version
3.7.1 (v3.7.1:260ec2c36a, Oct 20 2018, 14:05:16) [MSC v.1915 32 bit (Intel)]
NoneType: None
('python=3.7.1 (v3.7.1:260ec2c36a, Oct 20 2018, 14:05:16) [MSC v.1915 32 bit '

# Code 
mdf = MDF('C:/Temp/Delete/x2/JustSCR_InletTemp_Cval.mf4')
MyValues = mdf.get('SCR_InletTemp_Cval').samples

  ## MDF version
asammdf=5.1.0

  ## Code snippet
mdf = MDF('C:/Temp/Delete/x2/JustSCR_InletTemp_Cval.mf4')
MyValues = mdf.get('SCR_InletTemp_Cval').samples
  
  ## Traceback
Ok it's not really a traceback, this is from the console
 asammdf - ERROR - Expected "##TX" or "##MD" block @0x208 but found "b'##CC'"
  
# Description
Values are not correct
When importing the channel SCR_InletTemp_Cval  with
MyValues = mdf.get('SCR_InletTemp_Cval').samples
MyValues are weird, maybe scaled wrong

[JustSCR_InletTemp_Cval.zip](https://github.com/danielhrisca/asammdf/files/2946816/JustSCR_InletTemp_Cval.zip)
MyValues[10] = 9936 but it should be like 28 +- 1

I do get the ##TX error message and I believe it is linked to that channel. It does not stop the code. 

I get this message in the console, 
asammdf - ERROR - Expected "##TX" or "##MD" block @0x208 but found "b'##CC'"

Thank you for your help, Chris

**Comment by danielhrisca** [comment]
@tistelac 

you have a channel with partial conversion. 

Run this code to get the physical values

```python
from asammdf import MDF

mdf = MDF(filename)

ch = mdf.get('SCR_InletTemp_Cval').physical()
print(ch)
ch.plot()
```

**Comment by danielhrisca** [comment]
> I get this message in the console,
> asammdf - ERROR - Expected "##TX" or "##MD" block @0x208 but found "b'##CC'"

This is due to the partial conversion (instead of a TXBLOCK the conversion references another CCBLOCK), don't worry about it

**Comment by tistelac** [comment]
This is great, thank you. From now on I will use .physical() instead of .samples !

**Comment by danielhrisca** [comment]
I'm preparing a fix for this for release 5.1.1 that will properly respect the raw argument of get method. Method physical will be removed.

**Comment by danielhrisca** [comment]
@tistelac 
please check the code in the development branch. When you call _get_ with _raw=False_ you will get the converted values (should be the same as calling _physical_ in the previous implementation).

**Comment by danielhrisca** [comment]
@tistelac 
you can now use the released version 5.1.1
